package.json

```diff
@@ -141,6 +141,7 @@
     "faunadb": "2.6.1",
     "firebase": "7.14.5",
     "flat": "5.0.2",
+    "form-data": "4.0.0",
```

Comment:
This should be in the e2e test instead of here, right?

Reply:
Oh I see now, this was used by the test assertions (not the test fixture) so it probably needs to be here

Reply:
reverted 😄